csharp/ql/src/semmle/code/csharp/Property.qll

```diff
@@ -370,6 +374,10 @@ class Accessor extends Callable, Modifiable, Attributable, @callable_accessor {
   override Location getALocation() { accessor_location(this, result) }
 
   override string toString() { result = getName() }
+
+  override string getAPrimaryQlClass() {
+    result = "Accessor" and not this instanceof Getter and not this instanceof Setter
+  }
```

Comment:
I don't think this is needed as the other accessors are `AddEventAccessor` and `RemoveEventAccessor` which define their own `getAPrimaryQlClass()`.